### DISCUSSION TITLE
Is there a way to change the video thumbnail?

### DISCUSSION THREAD

**Discussion by millerbento** [Zulko/moviepy#1684] [Q&A]
Is there a way to **specify a frame or time to use as the video thumbnail** after exported?
I'm am exporting two versions of the same video, one is a shorter version of the same edit, so ideally I wanted them to have a different thumbnail so the user doesn't think that it is the same file duplicated.

I have already looked at the documentation but couldn't find anything related to this subject.
Any help will be much appreciated!

**Comment by keikoro** [comment]
@milexBH Please open an issue for your question, the Discussions section isn't really suited for what you're asking.
